test/extended/util/client.go

```diff
@@ -337,8 +352,160 @@ func (c *CLI) SetupProject() string {
 	return newNamespace
 }
 
+func (c *CLI) setupNamespace() string {
+	requiresTestStart()
+	newNamespace := names.SimpleNameGenerator.GenerateName(fmt.Sprintf("e2e-test-%s-", c.kubeFramework.BaseName))
+	username := fmt.Sprintf("%s-user", newNamespace)
+	serviceAccountName := "default"
+	c.SetNamespace(newNamespace)
+
+	nsObject := &corev1.Namespace{
```

Comment:
missing annotations https://github.com/openshift/openshift-apiserver/blob/master/pkg/project/apiserver/registry/projectrequest/delegated/sample_template.go#L46-L48

Reply:
Added them.

Reply:
@deads2k if these are not added in Microshift, why do we add them? The tests that require them should rightfully fail in our case, no?

Reply:
> @deads2k if these are not added in Microshift, why do we add them? The tests that require them should rightfully fail in our case, no?

Possibly, though I think we can say, "make an equivalent namespace"?  I could also get behind trying to tighten once we break the back of the problem.